app/javascript/components/Location.js

```diff
@@ -14,7 +14,7 @@ class Location {
     this.height                 = props.height                || 50;
     this.highlightColor         = null                        || '#FFB500';
     this.icon_height            = props.icon_height           || 50;
-    this.icon_id                = props.icon_id               || 0;
+    this.icon_id                = props.icon_id               || '';
```

Comment:
I set this as an empty string. Im not sure if 0 can be a valid id for an object. So to bypass that all together its a "blank string" which is much more robust.

Reply:
0 wouldn't ever be a valid ID because these would be database id's.